src/renderer/components/context.ts

```diff
@@ -5,7 +5,7 @@ import { namespaceStore } from "./+namespaces/namespace.store";
 export interface ClusterContext {
   cluster?: Cluster;
   allNamespaces?: string[]; // available / allowed namespaces from cluster.ts
-  contextNamespaces?: string[]; // selected by user (see: namespace-select.tsx)
+  selectedNamespaces?: string[]; // selected by user (see: namespace-select.tsx)
```

Comment:
I can revert this change if wanted. But `contextNamespaces` is a very not descriptive name IMO.